articles/api-auth/tutorials/authorization-code-grant-pkce.md

```diff
@@ -12,6 +12,15 @@ This tutorial will help you implement the Authorization Code (PKCE) grant. If yo
 
 The __Authorization Code with PKCE__ is the OAuth 2.0 grant that [native apps](/quickstart/native) use in order to access an API. In this document we will work through the steps needed in order to implement this: create a code verifier and a code challenge, get the user's authorization, get a token and access the API using the token.
 
+::: panel Prerequisites
+
+Before beginning this tutorial, please:
+
+* Set the Client's [Grant Type property](/clients/client-grant-types)
```

Comment:
I think the goal is to have all clients have a grant type. You're right in that it's not a must-do, so I changed it to a "check to see if..." kind of thing